02_bubble/brvishnu92/main_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBubble(t *testing.T) {
+	testCases := map[string]struct {
+		input    []int
+		expected []int
+	}{
+		"test1": {input: []int{5, 2, 3, 1}, expected: []int{1, 2, 3, 5}},
```

Comment:
add empty slice, single element and slice with multiple elements of same value please.

Reply:
Added these corner test cases 👍